app/templates/components/learner-select-field.hbs

```diff
@@ -0,0 +1,6 @@
+{{#if selection.value}}
```

Comment:
I don't like using a span and icon here when a generic HTML select tag will do the same job.  Is there a benefit to doing it this way?